fdbclient/KeyRangeMap.h

```diff
@@ -103,6 +103,7 @@ void krmSetPreviouslyEmptyRange( struct CommitTransactionRef& tr, Arena& trArena
 Future<Void> krmSetRange( Transaction* const& tr, Key const& mapPrefix, KeyRange const& range, Value const& value );
 Future<Void> krmSetRange( Reference<ReadYourWritesTransaction> const& tr, Key const& mapPrefix, KeyRange const& range, Value const& value );
 Future<Void> krmSetRangeCoalescing( Transaction* const& tr, Key const& mapPrefix, KeyRange const& range, KeyRange const& maxRange, Value const& value );
+Future<Void> krmSetRangeCoalescing( Reference<ReadYourWritesTransaction> const& tr, Key const& mapPrefix, KeyRange const& range, KeyRange const& maxRange, Value const& value );
```

Comment:
I forgot that we added this new overload of krmSetRangeCoalescing. I think it should just take a pointer like the other one, and then we don't need to do the whole "holdWhile" thing

Reply:
I made it as a Reference like what `krmSetRange` does to make the interface consistent.

Reply:
Ahh right. Fair enough.